src/main/java/com/backbase/ct/bbfuel/input/PocketsReader.java

```diff
@@ -0,0 +1,32 @@
+package com.backbase.ct.bbfuel.input;
+
+import com.backbase.ct.bbfuel.data.CommonConstants;
+import com.backbase.ct.bbfuel.util.CommonHelpers;
+import com.backbase.ct.bbfuel.util.ParserUtil;
+import com.backbase.dbs.pocket.tailor.client.v1.model.PocketPostRequest;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class PocketsReader extends BaseReader {
+
+    public PocketPostRequest loadSingle() {
+        return CommonHelpers
+            .getRandomFromList(load(globalProperties.getString(CommonConstants.PROPERTY_POCKETS_DATA_JSON)));
```

Comment:
please also add a property specifying the path to the pockets.json on data.properties file instead of hardcoding it on the code. So we can have multiple files in the case we need to test edge cases in the future.

Reply:
I will use a property